### PR TITLE
HZC-3667: Update README.md with env variable for Viridian

### PR DESCRIPTION
**Pull request by alaktionov**
image

### CODE REVIEW

**Review by sdreger — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,7 +42,15 @@ wget \
   -O /usr/local/bin/hzcloud && chmod +x /usr/local/bin/hzcloud
 ```
 ## Authentication with Hazelcast Cloud
-After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer).
+After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer).
```
Comment (kprzygudzki) [inline README.md]: some whitespace was added at the end of this line :harold:

File: README.md [hunk 2]
```diff
@@ -42,7 +42,15 @@ wget \
   -O /usr/local/bin/hzcloud && chmod +x /usr/local/bin/hzcloud
 ```
 ## Authentication with Hazelcast Cloud
-After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer).
+After a successful installation, in order to use, you need to authenticate with Hazelcast Cloud by providing access tokens, which can be created from `Developers` tab in [Hazelcast Cloud](https://cloud.hazelcast.com/settings/developer). You can check how to generate API Key and API Secret following the [Hazelcast Cloud Documentation](https://docs.cloud.hazelcast.com/docs/developer). 
+
+`hzcloud` is configured against [Hazelcast Cloud](https://cloud.hazelcast.com) by default. However, you can use it with [Hazelcast Viridian](https://viridian.hazelcast.com/) as well by setting `HZ_CLOUD_API_URL` environment variable:
```
Comment (kprzygudzki) [inline README.md]: As much as I think the word `against` is correct in this context, I still can imagine a situation where it be confusing for someone.
Maybe let's say something like `configured to use [Hazelcast Cloud]`?

File: README.md [hunk 3]
```diff
@@ -114,6 +114,14 @@ hzcloud enterprise-cluster list
 hzcloud version update
 ```
 
+### Experimental
+`hzcloud` configured to use [Hazelcast Cloud](https://cloud.hazelcast.com) by default. However, you can use it with [Hazelcast Viridian](https://viridian.hazelcast.com/) as well by setting `HZ_CLOUD_API_URL` environment variable (don't forget to get authenticated accordingly, see [Authentication with Hazelcast Cloud](#authentication-with-hazelcast-cloud) section above):
+```sh
+export HZ_CLOUD_API_URL=https://viridian.hazelcast.com/api/v1
```
Comment (hz-aborovenskyi) [inline README.md:120]: It doesn't look as a proper link.
I think the proper one is https://api.viridian.hazelcast.com

**Review by hz-aborovenskyi — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (alaktionov) [reply README.md:120]: It seems correct 
image

**Review by kprzygudzki — APPROVED** [review]

**Pull request merged**